app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -30,4 +30,7 @@ message Target {
     map<string, string> properties = 6;
     // Name of the LUCI builder to trigger.
     optional string builder = 7;
+    // Name of the scheduler to trigger this target.
+    // Defaults to being triggered by cocoon. Supports 'cocoon', 'luci'.
+    optional string scheduler = 8 [default = 'cocoon'];
```

Comment:
I"m guessing it's non-trivial to parse an enum out of a user-maintained yaml file?

Reply:
It is trivial. I didn't think of using an enum. Updated :-)